src/backend/updates.rs

```diff
@@ -80,80 +80,18 @@ fn post_upgrade() {
-            state
-                .memory
-                .posts
-                .reinsert(2, serde_json::from_value(post_2).unwrap());
-            state
-                .memory
-                .posts
-                .reinsert(3, serde_json::from_value(post_3).unwrap());
-        });
-
-        // Restore features
-        mutate(|state| state.memory.features.reset());
-        let mut features = Vec::new();
-        read(|state| {
-            for post_id in 1136641..state.next_post_id {
-                if let Some(post) = Post::get(&state, &post_id) {
-                    if !matches!(post.extension, Some(Extension::Feature)) {
-                        continue;
-                    }
-                    let author = state.user(post.user.to_string().as_str()).unwrap();
-                    features.push((author.principal, post_id));
-                }
-            }
-        });
-
-        features
-            .into_iter()
-            .for_each(|(author, post_id)| features::create_feature(author, post_id).unwrap());
-    }
-
-    // Fills new data structures with data
     mutate(|state| {
-        // create indexes for all tags of active subscribers
-        state.tag_indexes = state.users.values().fold(HashMap::new(), |mut acc, user| {
-            for tag in user.feeds.iter().flatten() {
-                let index = acc.entry(tag.clone()).or_default();
-                index.subscribers += 1;
-            }
-            acc
-        });
-
-        // create indexes for all tags
-        for (post_id, tag) in state
-            .posts_with_tags
-            .iter()
-            .filter_map(|id| Post::get(state, id))
-            .flat_map(|post| {
-                post.tags
-                    .iter()
-                    .map(move |tag| (post.id, tag.to_lowercase()))
-            })
-            .collect::<Vec<_>>()
-            .into_iter()
+        // Resetting the report of @KeydaKorven (/post/1219744)
         {
-            let entry = state.tag_indexes.entry(tag.clone()).or_default();
-            entry.posts.push_front(post_id);
-            while entry.posts.len() > 1000 {
-                entry.posts.pop_back();
-            }
-        }
-    });
-
-    // Compensate for the executed by failed proposal #/post/1159871
-    mutate(|state| {
-        if let Some(realm) = state.realms.get_mut("RUGANG") {
-            realm.controllers.insert(1277);
+            let _ = Post::mutate(state, &1180586, |post| {
+                post.report.take();
+                Ok(())
+            });
+            let Some(user) = state.users.get_mut(&3750) else {
```

Comment:
Did you mean 3570? https://6qfxa-ryaaa-aaaai-qbhsq-cai.ic0.app/#/user/3570
3750 is CristiNU, not KeydaKorven.

Nit: using `if let Some(user) {...}` seems more readable to me compared to `return`.

Reply:
Oh wow, thanks a lot Ulan! Hm, I was under the impression that I _copied_ the user id from the url...  😅 

Fixed!